.github/workflows/eas-build.yml

```diff
@@ -0,0 +1,57 @@
+name: EAS Build
+
+on:
+  push:
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  eas-build:
+    name: EAS Build
+    environment: EAS Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          fetch-depth: 0 # Required due to the weg Git works, without it this action won't be able to find any or the correct tags
+
+      - uses: "WyriHaximus/github-action-get-previous-tag@8656db79964f6757c940965c172d371ceedc8a7f"
+        id: previoustag
+        with:
+          fallback: 1.0.0
+
+      - uses: actions/github-script@441359b1a30438de65712c2fbca0abe4816fa667
+        id: version
+        with:
+          script: |
+            const semver = '${{steps.previoustag.outputs.tag}}'.replace('v', '')
+            if (semver.match(/^[0-9]+\.[0-9]+\.[0-9]+$/)) {
+              return semver
+            }
+
+            throw new Error('not semver')
```

Comment:
how about using time e.g. `0.0.2109301845` should we know the exact build time?

Reply:
Also not sure how the App Store or OS will react if there is duplicated versioning with the current method.

Reply:
For APK, need to uninstall the current app and install the new one. I think should be ok since it's just for testing purposes.

I'll try the build number if it will be accepted by standard-version